README.md

```diff
@@ -17,14 +17,13 @@ Please read our [contributing guide](https://napari.org/dev/developers/contribut
 ### quickstart: local setup
 * __Check the prerequisites__
     1. Create a clean Python environment (e.g., with conda).
-    1. In that environment, create a [development installation](https://napari.org/dev/developers/contributing/dev_install.html#dev-installation) of `napari` on your local machine, first forking and cloning the main `napari` project if you've not previously done so.
-    1. Fork this repository, and then clone your fork to your local machine. NB: you may want to name your fork e.g. `napari-docs` rather than just `docs`.
-    1. With your local clone's root folder as the working directory, and with the environment created in the first step activated, install the docs requirements with `python -m pip install -r requirements.txt`.
+    1. In that environment, create an editable `napari` installation with the `docs` dependency group and a Qt backend. For example, after first forking and cloning the main `napari` project if you've not previously done so, run `python -m pip install -e "[.pyqt, docs]"` from your `napari/napari` clone directory. This will use the default Qt backend.
```

Comment:
```suggestion
    1. In that environment, create an editable `napari` installation with the `docs` dependency group and a Qt backend. For example, after first forking and cloning the main `napari` project if you've not previously done so, run `python -m pip install -e ".[pyqt, docs]"` from your `napari/napari` clone directory. This will use the default Qt backend.
```

Reply:
Should we mention Python>3.9?

Reply:
Good catch, I added, using the python range sphinx var like in the installation guide.
Hopefully building docs works across all versions -- i've tested 311, 312, and 313 today.